### PR TITLE
Remove the error filed

### LINKED ISSUE #95: Unify stdout and stderr as a single stream
All CI systems join the stdout and stderr streams to show a single view. Let's follow their lead. The checker and student code should have stdout and stderr set to the same file stream, so maybe append `2>&1` to `cd /vagrant; ./script.sh` [here](https://github.com/vmck/vagrant-vmck/blob/master/submission/launch.sh#L16).

Let's keep the rest of the code as-is, we can remove the `stderr` field later, after we're happy with the setup.

(Leaving the issue in the `acs-interface` repo since `vagrant-vmck` should be independent from `acs-interface`; at some point we should move that logic away from it)

### PR DESCRIPTION
**Pull request by gmuraru**
Fixes #95 

Remove the field with the stderr output.
This PR should be merged alongside [this](https://github.com/vmck/vagrant-vmck/pull/41)

### CODE REVIEW

File: interface/views.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def done(request, pk):
     assert submission.verify_jwt(request.GET.get('token'))
 
     stdout = utils.decode(options['stdout'])
-    stderr = utils.decode(options['stderr'])
+    stderr = utils.decode(options['stderr'], '')
```
Comment (mgax) [inline interface/views.py]: You sure this works? [`decode`](https://github.com/vmck/acs-interface/blob/3683329573c9a063608f0675f5d3277449bd5dc0/interface/utils.py#L36) only accepts one argument.

File: interface/models.py [hunk 2]
```diff
@@ -154,12 +154,12 @@ def compute_review_score(self):
...
-        timestamp = self.timestamp.strftime(penalty.DATE_FORMAT)
-        deadline = self.assignment.deadline_soft.strftime(penalty.DATE_FORMAT)
+        timestamp = self.timestamp or datetime.datetime.now()
+        deadline = self.assignment.deadline_soft
 
         return penalty.compute_penalty(
-            timestamp,
-            deadline,
+            timestamp.strftime(penalty.DATE_FORMAT),
+            deadline.strftime(penalty.DATE_FORMAT),
```
Comment (mgax) [inline interface/models.py]: This bit should not be in this PR. Please `git fetch` and merge with `origin/master`.

Comment (gmuraru) [reply interface/models.py]: Updated

File: interface/views.py (hunk 1 above)
Comment (gmuraru) [reply interface/views.py]: Removed the stderr from the ```views``` and ```templates```

**Comment by gmuraru** [comment]
@mgax in order to test this I need [this] (https://github.com/vmck/vagrant-vmck/pull/41) merged, right?

**Comment by mgax** [comment]
> @mgax in order to test this I need [this] ([vmck/vagrant-vmck#41](https://github.com/vmck/vagrant-vmck/pull/41)) merged, right?

You can make your own docker build of `vagrant-vmck`, publish it under your docker account, and configure vmck to use your image.

As a side note, vmck can be configured to use another image with `MANAGER_TAG=0.3.2`, but the repo is hardcoded. So, bonus points if you modify vmck, to allow the entire docker image name to be configured, somewhere around [here](https://github.com/vmck/vmck/blob/bc0acbd21b88c1755686b923e7b759e70296b02e/vmck/backends/submission.py#L9) :)

**Comment by mgax** [comment]
Actually, you can test with the regular `vagrant-vmck` image, it will send `stderr` but that will get ignored.

**Pull request merged**